Source/Filter Effects/feColourMatrix/SvgColourMatrix.cs

```diff
@@ -87,7 +87,7 @@ public override void Process(ImageBuffer buffer)
                     };
                     break;
                 default: // Matrix
-                    var parts = this.Values.Replace("  ", " ").Split(new char[] { ' ', '\t', '\n', '\r', ',' });
+                    var parts = this.Values.Split(new char[] { ' ', '\t', '\n', '\r', ',' }, StringSplitOptions.RemoveEmptyEntries);
                     colorMatrixElements = new float[5][];

```

Comment:
Thanks - good catch! Could you add your test svg together with the corresponsing png as a regression test?

Reply:
I can do this but later if that's ok?

Reply:
Sure!

Reply:
@mrbean-bremen Seems like the png is not rendered correctly using SVG lib. The issue was only related to the string parsing not rendering.

Reply:
![SVGViewer_2019-08-04_22-03-26](https://user-images.githubusercontent.com/2297442/62428613-bc183a00-b703-11e9-8341-be9f1811ad8b.png)

Reply:
The output from my renderer:
![element-feColorMatrix](https://user-images.githubusercontent.com/2297442/62428652-131e0f00-b704-11e9-9ee6-1ca4312cd31d.png)

You can see one issue, the `Filter` property is inherited by Children of the `use` element, which is not what spec says. The `Fileter` property is not inheritable: https://www.w3.org/TR/SVG11/filters.html#FilterProperty, the same issue I had was with `Opacity`: https://www.w3.org/TR/SVG11/masking.html#ObjectAndGroupOpacityProperties

Reply:
Thanks for digging into this! I will merge as is in this case. Filters are generally very poorly supported (see #43), so I'm not too surprised by this, though the inheritance problem is something that certainly shall be fixed.